trustpilot/cli.py

```diff
@@ -19,36 +19,61 @@ def get_verbosity(ctx):
     return ctx.meta.get("trustpilot.verbosity", 0)
 
 
+@click.pass_context
+def get_output_format(ctx):
+    output_format = ctx.meta.get("trustpilot.outputformat", "json") or "json"
+    return output_format
+
+
 def format_response(response):
     content = response.text
-    try:
-        content = response.json()
-    except ValueError:
-        pass
-    output = OrderedDict()
-    output["url"] = response.url
-    output["status"] = response.status_code
-    if get_verbosity():
-        headers = response.headers
-        output["headers"] = OrderedDict((k, headers[k]) for k in headers)
-    output["content"] = content
 
-    return json.dumps(output, indent=2)
+    output_format = get_output_format()
+    if output_format == "raw":
```

Comment:
Would it make sense to put some time into writing some basic tests around the expected formats. Especially since this is a very manual process of writing lines perhaps there should be some regressions tests.

Reply:
Which would also push for making these two formatters into their own functions which narrows down the scope needed to processes this specific code here.

Reply:
yeah, good ideas but i dont think we have the time to prioritize this right now